requirements-dev.txt

```diff
@@ -0,0 +1,6 @@
+pytest==2.8.3
```

Comment:
add `-r requirements.txt` so to avoid having to include the default requirements